src/main/java/seedu/address/logic/commands/DeletePersonCommand.java

```diff
@@ -60,13 +70,14 @@ public boolean equals(Object other) {
         }
 
         DeletePersonCommand otherDeleteCommand = (DeletePersonCommand) other;
-        return targetIndex.equals(otherDeleteCommand.targetIndex);
+
+        return targetNusNet.equals(otherDeleteCommand.targetNusNet);
     }
 
     @Override
     public String toString() {
         return new ToStringBuilder(this)
-                .add("targetIndex", targetIndex)
+                .add("targetNUSNET_ID", targetNusNet)
```

Comment:
Consider sticking to the same format:
```suggestion
                .add("targetNusNet", targetNusNet)
```